cypress/support/pages/laptopPage.cy.js

```diff
@@ -0,0 +1,31 @@
+class ClickOnTheLaptops {
+  selectLaptopsCategory() {
+    cy.contains('Laptops').click();
+  }
+
+  selectSonyLaptop() {
+    cy.get('[class="hrefch"]')
+      .contains('Sony vaio i7')
+      .click();
+  }
+
+  clickOnTheAddBtn() {
+    cy.get('[class="btn btn-success btn-lg"]')
+      .contains('Add to cart').click();
+    cy.on('window:alert', (str) => {
+      expect(str).to.contains('Product added');
+    });
+    cy.on('window:confirm', () => true);
+  }
+
+  clickOnTheOrderBtn() {
```

Comment:
according to the name of your method, you need to click on the order btn, but you have the click on cart button also. Create a new method for clicking the cart button action